tests/test_codemod.py

```diff
@@ -330,6 +330,17 @@ def __exit__(self, a, b, c, d):
         """
         self.assertCodemod(before, after, annotate_magics=True)
 
+    def test_zip_star(self) -> None:
```

Comment:
Rename the test to something like `test_empty_elems` and make the function body empty.